etc/containerpilot.json

```diff
@@ -8,7 +8,16 @@
       "port": 80,
       "health": "/usr/bin/curl --fail --silent --show-error --output /dev/null http://localhost/nginx-health",
       "poll": 10,
-      "ttl": 25
+      "ttl": 25,
+      "interfaces": ["eth0"]
+    },
+    {
+      "name": "nginx-public",
```

Comment:
The line we changed in the existing `nginx` service just makes the default behavior explicit. The only potential consumer of `nginx-public` right now is something like `autopilotpattern/cloudflare` (which has its own nginx). Generally we don't have consumers of the public IP service because those are outside the deployment.